main.c

```diff
@@ -1,50 +1,25 @@
 #include <stdio.h>
 #include <assert.h>
 
-enum MajorColor {WHITE, RED, BLACK, YELLOW, VIOLET};
-enum MinorColor {BLUE, ORANGE, GREEN, BROWN, SLATE};
-
-const char* MajorColorNames[] = {
-    "White", "Red", "Black", "Yellow", "Violet"
-};
-int numberOfMajorColors =
-    sizeof(MajorColorNames) / sizeof(MajorColorNames[0]);
-const char* MinorColorNames[] = {
-    "Blue", "Orange", "Green", "Brown", "Slate"
-};
+#include "colorpair.h"
+const char* MajorColorNames[] = {"White", "Red", "Black", "Yellow", "Violet"};
```

Comment:
definition of related enum and constants spread in different files. Logical files where this can be kept in are colorpair.h/.c